.ci/Jenkinsfile

```diff
@@ -124,34 +124,44 @@ pipeline {
         HOME = "${env.WORKSPACE}"
         PATH = "${env.PATH}:${env.WORKSPACE}/.local/bin"
       }
-      input {
-        message 'Should we release a new version?'
-        ok 'Yes, we should.'
-        parameters {
-          choice(
-            choices: [
-              'https://upload.pypi.org/legacy/',
-              'https://test.pypi.org/legacy/'
-             ],
-             description: 'PyPI repository URL',
-             name: 'REPO_URL')
-        }
-      }
       when {
-        beforeAgent true
         beforeInput true
         anyOf {
           tag pattern: 'v\\d+.*', comparator: 'REGEXP'
           expression { return params.Run_As_Master_Branch }
         }
       }
-      steps {
-        withGithubNotify(context: 'Release') {
-          deleteDir()
-          unstash 'source'
-          unstash('packages')
-          dir("${BASE_DIR}"){
-            releasePackages()
+      stages {
+        stage('Notify') {
+          steps {
+              emailext subject: '[apm-agent-python] Release ready to be pushed',
+                       to: 'observability-team@elastic.co',
```

Comment:
I'd use the email in NOTIFY_TO credential, is where we sent the build messages, this list if for other purposes we do not want to spam it

Reply:
This notification happens for the releases, no strong opinion whether to use `NOTIFY_TO` or `observability-team@elastic.co`, @beniwohli, any concerns if we use the default build mailing list for the releases?